pyproject.toml

```diff
@@ -65,9 +65,14 @@ pyspark = { version = ">=3,<4", optional = true }
 regex = { version = ">=2021.7.6", optional = true }
 requests = { version = ">=2,<3", optional = true }
 Shapely = { version = ">=1.6,<1.8|>=1.9,<2", optional = true }
+snowflake-connector-python = { version = ">=2.7.10,<3", optional = true, extras = [
+  "pandas"
+] }
+snowflake-sqlalchemy = { version = ">=1.4.1,<2", optional = true, extras = [
+  "pandas"
+] }
```

Comment:
Can you add a comment about why we have to do this crazy business?

Reply:
Which part? Adding the transitive dependency, or something else?

Reply:
Done!